libs/net/keycloak/ClientRole.cs

```diff
@@ -0,0 +1,25 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace TNO.Keycloak;
+
+/// <summary>
+/// ClientRole enum, provides the possible Keycloak client roles.
+/// </summary>
+public enum ClientRole
```

Comment:
This is a hardcoded list of client roles that must be created in Keycloak for the solution to work.